taca/analysis/analysis_nanopore.py

```diff
@@ -44,13 +44,13 @@ def check_ongoing_sequencing(run_dir):
     else:
         return False
 
-def process_minion_run(minion_run, sequencing_ongoing=False, nanoseq_ongoing=False): 
-    """Process minion runs.
+def process_minion_run(minion_run, sequencing_ongoing=False, nanoseq_ongoing=False):
+    """Process MinION QC runs.
     
     Will not start nanoseq if a sequencing run is ongoing, to limit memory usage.
     Will also maximum start one nanoseq run at once, for the same reason.
     """
-    logger.info('Processing run: {} as a {}'.format(minion_run.run_dir, 'QC run' if minion_run.qc_run else 'non-QC run'))
+    logger.info('Processing run: {}'.format(minion_run.run_dir))
```

Comment:
Maybe the log can be `Processing QC run`? Don't change it if you think it's unnecessary.

Reply:
Yes, that's a good idea! I'll change it 👍